doc/requirements.rst

```diff
@@ -54,25 +54,29 @@ Motor and PyMongo
 +-------------------+-----------------+
 | 2.0               | 3.7+            |
 +-------------------+-----------------+
+| 2.1               | 3.9+            |
++-------------------+-----------------+
 
 Motor and MongoDB
 `````````````````
 
-+---------------------------------------------------------+-----+-----+
-|                    MongoDB Version                      |     |     |
-+=====================+=====+=====+=====+=====+=====+=====+=====+=====+
-|                     | 2.2 | 2.4 | 2.6 | 3.0 | 3.2 | 3.4 | 3.6 | 4.0 |
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-| Motor Version | 1.0 |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.1 |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.2 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.3 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 2.0 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+
++---------------------------------------------------------+-----+-----+-----+
+|                    MongoDB Version                      |     |     |     |
++=====================+=====+=====+=====+=====+=====+=====+=====+=====+=====+
+|                     | 2.2 | 2.4 | 2.6 | 3.0 | 3.2 | 3.4 | 3.6 | 4.0 | 4.2 |
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+| Motor Version | 1.0 |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.1 |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.2 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.3 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 2.0 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
```

Comment:
@ajdavis any idea why we say 2.6 is not supported by Motor, when it is definitely supported by the PyMongo versions these Motor releases use?

Reply:
Interesting. I decided this deliberately but I can't remember why:

https://jira.mongodb.org/browse/MOTOR-252

Reply:
Let's leave it be for now. I doubt anyone is using ancient Mongo with a cutting-edge driver.